src/components/CytoscapeGraph/CytoscapeGraph.tsx

```diff
@@ -147,15 +148,11 @@ export class CytoscapeGraph extends React.Component<CytoscapeGraphProps, Cytosca
   };
 
   private turnEdgeLabelsTo = (cy: any, value: EdgeLabelMode) => {
-    cy.edges().forEach(edge => {
-      edge.data('edgeLabelMode', value);
-    });
+    cy.scratch(CytoscapeGlobalScratchNamespace).edgeLabelMode = value;
```

Comment:
We probably don't need this one anymore

Reply:
this is called on line 330 - we need this to let GraphStyles know how to render the edge label -- see GraphStyles line 142

Reply:
Yes, but is already set on line 290 with exactly the same value (`this.props.edgeLabelMode`)